### DISCUSSION TITLE
StreamForgetting Metric plugin

### DISCUSSION THREAD

**Discussion by Mattdl** [ContinualAI/avalanche#460] [Feature Request] [answered]
Hi, I was wondering if there is a StreamForgetting similar to the StreamAccuracy plugin available?
So far I've only found the ExperienceForgetting. StreamForgetting would average over all the ExperienceForgetting Metrics.
This is commonly used when just having a single number besides the StreamAccuracy, so I think this would be helpful (:
Thanks!

**Comment by AndreaCossu** [comment]
No, currently we don't have it, but I agree that it could be useful, especially when dealing with many experiences :smile: Feel free to open an issue about it and if you want to try and implement it you can also self assign it.

**Reply by Mattdl** [reply]
Ok thanks for the reply! (:

**Comment by vlomonaco** [comment, marked as answer]
Done in #471.
